src/Themes/ui/Theme.tsx

```diff
@@ -66,6 +72,9 @@ export function refreshTheme(): void {
       successdark: Settings.theme.successdark,
       white: Settings.theme.white,
       black: Settings.theme.black,
+      mapstatic: Settings.theme.currentcity,
```

Comment:
You messed this up

Reply:
Ah, thanks for the review!  Turns out I'm removing `mapstatic` anyway, so this embarrassing mistake will be cast into the void.  Or maybe the null.